docs/_layouts/default.html

```diff
@@ -5,12 +5,19 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width">
 
-{% seo %}
+    {% seo %}
     <link rel="stylesheet" href="{{ "/assets/css/style.css?v=" | append: site.github.build_revision | relative_url }}">
     <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js"></script>
     <![endif]-->
-        </head>
+       <link href='https://fonts.googleapis.com/css?family=Barlow Semi Condensed' rel='stylesheet'>
+        <style>
+        body {
```

Comment:
The font change should be done in the CSS stylesheet not in the layout.

Reply:
(Background: As a general rule, it's alway better in the long run, mainly for maintainability and reusablity, to separate content (in our case .md), structure (the html layouts), and style (the style.scss))